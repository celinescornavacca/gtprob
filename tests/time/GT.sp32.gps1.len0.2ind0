((((AA0,((KA0,(QA0,RA0)),JA0)),((CA0,(((WA0,XA0),SA0),(EB0,FB0))),((NA0,OA0),(CB0,DB0)))),(((BA0,(GA0,HA0)),(IA0,(TA0,UA0))),(((AB0,BB0),FA0),(((LA0,MA0),EA0),DA0)))),(PA0,((YA0,ZA0),VA0)));
