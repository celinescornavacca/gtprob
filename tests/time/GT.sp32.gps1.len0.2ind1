((((((HA0,(EB0,FB0)),((MA0,(XA0,YA0)),(JA0,KA0))),CA0),AA0),(((((PA0,QA0),IA0),((VA0,WA0),(AB0,BB0))),DA0),(FA0,GA0))),(((NA0,OA0),EA0),((((RA0,SA0),(ZA0,(CB0,DB0))),(LA0,(TA0,UA0))),BA0)));
