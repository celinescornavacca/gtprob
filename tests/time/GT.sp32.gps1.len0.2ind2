(((AA0,(YA0,ZA0)),(((GA0,(VA0,(WA0,XA0))),(PA0,QA0)),(HA0,(JA0,(EB0,FB0))))),((((DA0,(NA0,OA0)),(BA0,CA0)),(EA0,(((KA0,LA0),(IA0,(((RA0,SA0),MA0),(TA0,UA0)))),FA0))),((CB0,DB0),(AB0,BB0))));
