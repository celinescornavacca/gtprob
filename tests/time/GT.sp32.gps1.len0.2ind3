(((HA0,IA0),((UA0,VA0),((CB0,DB0),OA0))),((((EB0,FB0),NA0),((((RA0,SA0),((WA0,XA0),(YA0,ZA0))),CA0),(EA0,(KA0,(LA0,MA0))))),((BA0,(DA0,(((TA0,(AB0,BB0)),JA0),(FA0,((PA0,QA0),GA0))))),AA0)));
