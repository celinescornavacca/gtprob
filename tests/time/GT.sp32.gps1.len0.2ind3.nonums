(((HA,IA),((UA,VA),((CB,DB),OA))),((((EB,FB),NA),((((RA,SA),((WA,XA),(YA,ZA))),CA),(EA,(KA,(LA,MA))))),((BA,(DA,(((TA,(AB,BB)),JA),(FA,((PA,QA),GA))))),AA)));
