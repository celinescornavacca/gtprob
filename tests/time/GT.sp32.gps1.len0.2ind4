(((((GA0,((SA0,TA0),(IA0,(VA0,WA0)))),EA0),((FA0,(LA0,MA0)),((((QA0,RA0),PA0),NA0),(EB0,FB0)))),((((ZA0,(CB0,DB0)),OA0),((AB0,BB0),UA0)),AA0)),((((((JA0,KA0),HA0),DA0),CA0),BA0),(XA0,YA0)));
