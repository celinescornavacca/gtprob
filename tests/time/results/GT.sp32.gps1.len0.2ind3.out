-35.3371459239
